Only Cleanup

vp-stage opened the pull request:
cleanup && organize imports

vp-stage committed:
+ cleanup && organize imports

vp-stage committed:
+ change WiremockResolver server field to protected in order to improve coverage.

vp-stage commented:
Hey @lanwen, 
you could be interested by this PR.

Kind regards

lanwen commented on src/test/java/ru/lanwen/wiremock/testlib/TicketEndpoint.java:
```diff
@@ -25,7 +22,7 @@
         ExtensionContext context = customizationContext.getExtensionContext();
         String uuid = UUID.randomUUID().toString();
-        String testMethodName = context.getTestMethod().get().getName();
+        String testMethodName = context.getTestMethod().orElseThrow(IllegalStateException::new).getName();
```
doesn't make any sense to replace one with another

lanwen commented on src/main/java/ru/lanwen/wiremock/ext/WiremockResolver.java:
```diff
@@ -40,15 +36,13 @@ public WiremockResolver() {
 
     @Override
-    public void afterEach(ExtensionContext testExtensionContext) throws Exception {
-        if (server == null || !server.isRunning()) {
```
fail fast is preferrable

lanwen commented on src/main/java/ru/lanwen/wiremock/ext/WiremockResolver.java:
```diff
@@ -29,7 +25,7 @@
 
     private final WiremockFactory wiremockFactory;
-    private WireMockServer server;
+    protected WireMockServer server;
```
👍

lanwen reviewed (changes requested):
star imports are banned, `final` doesn't add any value, fail fast is preferrable

vp-stage committed:
fix:
 - star imports are banned, final doesn't add any value, fail fast is preferable
 - Junit5 said:  test classes and methods do not have to be public anymore.

vp-stage commented:
Hey @lanwen,
the requested changes are done :)
